electron-component/main.js

```diff
@@ -0,0 +1,35 @@
+const WindowsManager = require("./components/WindowsManager");
+
+const electron = require('electron');
+const conf = require('./config');
+
+// Module to control application life.
+const app = electron.app;
+
+function initializeCommunicator() {
+    //the way to init the communicator instance:
+    require('./components/Communicator');
```

Comment:
1. do not require file inside your file, only in the file header
2. it is not a good practice to assume that requiring file will do something, always have an "init" method
3. remove comment

Reply:
done